packages/flutter/lib/src/widgets/fade_in_image.dart

```diff
@@ -254,6 +260,20 @@ class FadeInImage extends StatefulWidget {
   /// also affected by the scale factor.
   final double? width;
 
+  /// If non-null, this color is blended with each image pixel using [colorBlendMode].
+  /// Color applies to both [placeholder] and [image]
```

Comment:
This should probably only apply to the image. We'd need a separate property for the placeholder image, similar to how there is `filterQuality` and `placeholderFilterQuality`.